css/src/atomics/typography.scss

```diff
@@ -201,3 +201,29 @@
 .line-height-normal {
 	line-height: $line-height-normal;
 }
+
+// White space
+
```

Comment:
It's nice to have full coverage, but there's a size cost to including everything. Can we pare these down to include fewer options?

Reply:
can we start with these four for now? 

<img width="220" alt="image" src="https://github.com/user-attachments/assets/6aaf9c4d-2b02-4b71-9c15-0ef0b37fc7f7">